src/packages/client/src/__tests__/integration/happy/long-running-transaction/test.ts

```diff
@@ -0,0 +1,374 @@
+import { getTestClient } from '../../../../utils/getTestClient'
+
+let PrismaClient, prisma
+
+describe('long-running transaction', () => {
+  /**
+   * Minimal example of a long-running transaction
+   */
+  test('basic', async () => {
+    const result = await prisma.$transaction(async (prisma) => {
+      await prisma.user.create({
+        data: {
+          email: 'user_1@website.com',
+        },
+      })
+
+      await prisma.user.create({
+        data: {
+          email: 'user_2@website.com',
+        },
+      })
+
+      return prisma.user.findMany()
+    })
+
+    expect(result.length).toBe(2)
+  })
+
+  /**
+   * Transactions should fail after the default timeout
```

Comment:
There doesn't seem to be a test for the maxWait

Reply:
Ok, added to my todo list.